x/group/spec/01_concepts.md

```diff
@@ -1,4 +1,8 @@
-# Group Module
+<!--
```

Comment:
In the top of this file, there's `It does have an "administrator" which has the weight to add, remove and update members in the group.`

Is weight here correct? Shouldn't it be "power" or "ability" or something like that?